tests/sku-test.test.ts

```diff
@@ -1,11 +1,15 @@
-import { describe, it } from 'vitest';
+import { describe, it, afterEach } from 'vitest';
 
-import { scopeToFixture, waitFor } from '@sku-private/testing-library';
+import { scopeToFixture, waitFor, cleanup } from '@sku-private/testing-library';
 
 const { render } = scopeToFixture('sku-test');
 
 const testRunners = ['vitest', 'jest'] as const;
 
+afterEach(async () => {
+  await cleanup();
+});
+
```

Comment:
I'm not sure we're getting this for free with `vitest` because we're not injecting global, so I added a [cleanup](https://cli-testing.com/guides/api/#cleanup) call after each test.